url882f91ec97_WenboCao_Microsoft_Drone_tgz-pJ8-org_ejml_alg_dense_decomposition_qr_StabilityQRDecompositionJ8/src/org/ejml/data/FixedMatrix6x6_64F.java

```diff
@@ -0,0 +1,340 @@
+    }
+
+    @Override
+    public double unsafe_get(int row, int col) {
+        if( row == 0 ) {
+            if( col == 0 ) {
+                return a11;
+            } else if( col == 1 ) {
+                return a12;
+            } else if( col == 2 ) {
+                return a13;
+            } else if( col == 3 ) {
+                return a14;
+            } else if( col == 4 ) {
+                return a15;
+            } else if( col == 5 ) {
+                return a16;
+            }
+        } else if( row == 1 ) {
+            if( col == 0 ) {
+                return a21;
+            } else if( col == 1 ) {
+                return a22;
+            } else if( col == 2 ) {
+                return a23;
+            } else if( col == 3 ) {
+                return a24;
+            } else if( col == 4 ) {
+                return a25;
+            } else if( col == 5 ) {
+                return a26;
+            }
+        } else if( row == 2 ) {
+            if( col == 0 ) {
+                return a31;
+            } else if( col == 1 ) {
+                return a32;
+            } else if( col == 2 ) {
+                return a33;
+            } else if( col == 3 ) {
+                return a34;
+            } else if( col == 4 ) {
+                return a35;
+            } else if( col == 5 ) {
+                return a36;
+            }
+        } else if( row == 3 ) {
+            if( col == 0 ) {
+                return a41;
+            } else if( col == 1 ) {
+                return a42;
+            } else if( col == 2 ) {
+                return a43;
+            } else if( col == 3 ) {
+                return a44;
+            } else if( col == 4 ) {
+                return a45;
+            } else if( col == 5 ) {
+                return a46;
+            }
+        } else if( row == 4 ) {
+            if( col == 0 ) {
+                return a51;
+            } else if( col == 1 ) {
+                return a52;
+            } else if( col == 2 ) {
+                return a53;
+            } else if( col == 3 ) {
+                return a54;
+            } else if( col == 4 ) {
+                return a55;
+            } else if( col == 5 ) {
+                return a56;
+            }
+        } else if( row == 5 ) {
+            if( col == 0 ) {
+                return a61;
+            } else if( col == 1 ) {
+                return a62;
+            } else if( col == 2 ) {
+                return a63;
+            } else if( col == 3 ) {
+                return a64;
+            } else if( col == 4 ) {
+                return a65;
+            } else if( col == 5 ) {
+                return a66;
+            }
+        }
+        throw new IllegalArgumentException("Row and/or column out of range. "+row+" "+col);
+    }
+
+    @Override
+    public void set(int row, int col, double val) {
+        unsafe_set(row,col,val);
+    }
+
+    @Override
+    public void unsafe_set(int row, int col, double val) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 43. By comparison, 99% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.